### DISCUSSION TITLE
Export HTML with custom template

### DISCUSSION THREAD

**Discussion by Viguro79** [Zettlr/Zettlr#3362] [Q&A] [answered]
Well, I can't understand how to html export my document using a custom css. The default one is basic and I'd like to have a more "github" rendering.
I though I could use [this one css file](https://gist.github.com/killercup/5917178) in order to do that. Downloaded it on my window computer ; then in zettlr assets manager, I added the full path to this file in the html conf template, using the `css` variable.
`css: [<c:\\path to download folder\pandoc.css]
`
But the exported html file doesn't use this css. In fact, it doesn't even use the default one, and the html generated is very basic. So I assume something is wrong but what ? How can I use a custom css file on windows to generate my html file ?
Thanks!

**Comment by Viguro79** [comment, marked as answer]
I did something wrong. It's working now.

**Comment by jorg03** [comment]
@Viguro79 
Could you please describe your solution. I am trying the same and do not not how.
Thanks!

**Reply by Viguro79** [reply]
I download the file on my computer. Then in zettlr assets manager, I added the full path to this file in the html conf template, using the css variable.
css: [<c:\\path to download folder\pandoc.css]
Then I did an html export. And that's it !

**Reply by jorg03** [reply]
I found my mistake. Now it works. Thanks!

**Reply by bkuz114** [reply]
In case it's helpful to anyone.. there is a typo in the answer that was given:

` css: [<c:\path to download folder\pandoc.css]`

is missing the right caret. Either way, make sure that you do not include left or right carets in your css var; they were putting them in the answer to demonstrate that what's there should be substituted for your value. i.e. it should look something like this:

` css: [c::\mypath\pandoc.css]`

Thank you Virguro79 for posting this question + answer - it helped me solve my zettlr problem.
